### PR TITLE
Map: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/Map/00 Overview.md [hunk 1]
```diff
@@ -44,6 +44,62 @@ The code below adds the **Map** widget to your page. The **Map** is [centered](/
...
+        constructor(props) {
+            super(props);
+        }
+
+        render() {
+            return (
+                <Map
+                    zoom={10}
+                    controls={true}
+                    center={centerCoordinates}
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Map/00 Overview.md]:
```suggestion
                    defaultCenter={centerCoordinates}
```

File: concepts/05 Widgets/Map/00 Overview.md [hunk 2]
```diff
@@ -44,6 +44,62 @@ The code below adds the **Map** widget to your page. The **Map** is [centered](/
...
+
+    import { Map } from 'devextreme-react/map';
+
+    const centerCoordinates = { lat: 40.749825, lng: -73.987963 };
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+        }
+
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Map/00 Overview.md]:
```suggestion

```

Comment (RomanTsukanov) [reply concepts/05 Widgets/Map/00 Overview.md]: Please make the same change to all examples in which the constructor is unused.

File: concepts/05 Widgets/Map/00 Overview.md [hunk 3]
```diff
@@ -44,6 +44,62 @@ The code below adds the **Map** widget to your page. The **Map** is [centered](/
...
+        render() {
+            return (
+                <Map
+                    zoom={10}
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Map/00 Overview.md]:
```suggestion
                    defaultZoom={10}
```

Comment (RomanTsukanov) [reply concepts/05 Widgets/Map/00 Overview.md]: Please make the same change in all examples in which `autoAdjust` is not set or set to `true`.

File: concepts/05 Widgets/Map/00 Overview.md (hunk 1 above)
Comment (RomanTsukanov) [reply concepts/05 Widgets/Map/00 Overview.md]: Please make the same change in all examples.

File: concepts/05 Widgets/Map/10 Specify the Provider and Type.md [hunk 4]
```diff
@@ -85,6 +139,74 @@ When using maps, you should include an API key that authenticates your applicati
...
+        components: {
+            DxMap
+        },
+        data() {
+            return {
+                centerCoordinates: { lat: 40.749825, lng: -73.987963 },
+                authentificationKeys: {
+                    bing: this.BING_API_KEY,
+                    google: this.GOOGLE_API_KEY,
+                    googleStatic: this.GOOGLE_STATIC_API_KEY
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Map/10 Specify the Provider and Type.md]: These variables have to be declared somewhere.

File: concepts/05 Widgets/Map/10 Specify the Provider and Type.md [hunk 5]
```diff
@@ -85,6 +139,74 @@ When using maps, you should include an API key that authenticates your applicati
...
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Map } from 'devextreme-react/map';
+
+    const centerCoordinates = { lat: 40.749825, lng: -73.987963 };
+    const authentificationKeys = {
+        bing: this.BING_API_KEY,
+        google: this.GOOGLE_API_KEY,
+        googleStatic: this.GOOGLE_STATIC_API_KEY
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Map/10 Specify the Provider and Type.md]: - These variables have to be declared somewhere.
- What does `this` refer to in this case?

File: concepts/05 Widgets/Map/20 Configure Markers/10 Customize.md [hunk 6]
```diff
@@ -44,6 +44,72 @@ The **Map** widget allows you to provide a single icon for all markers. For this
...
+    import { DxMap } from 'devextreme-vue/map';
+
+    export default {
+        components: {
+            DxMap
+        },
+        data() {
+            return {
+                markerIconUrl: "https://js.devexpress.com/Demos/RealtorApp/images/map-marker.png",
+                mapMarkers = [
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Map/20 Configure Markers/10 Customize.md]:
```suggestion
                mapMarkers: [
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Map/00 Overview.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/Map/00 Overview.md]: fixed

File: concepts/05 Widgets/Map/00 Overview.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/Map/00 Overview.md]: fixed

File: concepts/05 Widgets/Map/00 Overview.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/Map/00 Overview.md]: fixed

File: concepts/05 Widgets/Map/20 Configure Markers/10 Customize.md (hunk 6 above)
Comment (ksercs) [reply concepts/05 Widgets/Map/20 Configure Markers/10 Customize.md]: fixed

File: concepts/05 Widgets/Map/10 Specify the Provider and Type.md (hunk 5 above)
Comment (ksercs) [reply concepts/05 Widgets/Map/10 Specify the Provider and Type.md]: - see the thread above
- `this` is excess here. Fixed

File: concepts/05 Widgets/Map/10 Specify the Provider and Type.md (hunk 4 above)
Comment (ksercs) [reply concepts/05 Widgets/Map/10 Specify the Provider and Type.md]: We decided to change the API keys placeholders to strings, as in the [demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Map/ProvidersAndTypes/React/Light/)

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**